tests/Mutant/Generator/MutationsGeneratorTest.php

```diff
@@ -107,50 +91,6 @@ public function test_it_can_skip_not_covered_on_file_level(): void
         $this->assertCount(0, $mutations);
     }
 
-    public function test_it_can_skip_not_covered_on_file_line_level(): void
```

Comment:
These 2 tests are failing, and i can't really figure out how they are working.

The problem with most of the tests here is that they are mocking dependencies of dependencies, which makes it difficult to figure out what is going on.

It is also testing behavior of child classes, instead of things it does itself.